packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/mediaconvert/create-job.ts

```diff
@@ -0,0 +1,105 @@
+import { Construct } from 'constructs';
+import * as iam from '../../../aws-iam';
+import * as sfn from '../../../aws-stepfunctions';
+import { Stack } from '../../../core';
+import * as cdk from '../../../core';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for creating a MediaConvert Job
+ *
+ * See the CreateJob API for complete documentation
+ * @see https://docs.aws.amazon.com/mediaconvert/latest/apireference/jobs.html#jobspost
```

Comment:
Is this link correct? I opened this link, but not sure if it's related to the create job api.

Reply:
Yes, the link https://docs.aws.amazon.com/mediaconvert/latest/apireference/jobs.html#jobspost talks about the POST Operation CREATEJOB API that creates a new transcoding job in MediaConvert service.